src/back/TicketManager/src/Core/TicketManager.Core.Contracts/Organizers/OrganizerDecideRequest.cs

```diff
@@ -0,0 +1,13 @@
+namespace TicketManager.Core.Contracts.Organizers;
+
+public class OrganizerDecideRequest
+{
+    public Guid OrganizerId { get; set; }
+    public bool IsAccepted { get; set; }
+
+    public static class ErrorCodes
+    {
+        public static int OrganizerNotInDatabase = 1;
```

Comment:
Maybe `OrganizerDoesNotExist`?
It's less implementation related. `OrganizerNotInDatabase` wouldn't make any sense, if we didn't have database. Client doesn't have to know that we have database.